test_outputs/query_execution/declarative_macro_missing.snap

```diff
@@ -0,0 +1,18 @@
+---
+source: src/query.rs
+expression: "&query_execution_results"
+---
+{
+  "./test_crates/declarative_macro_missing/": [
+    {
+      "name": String("will_be_removed"),
+      "span_begin_line": Uint64(2),
+      "span_filename": String("src/lib.rs"),
+    },
+    {
+      "name": String("will_no_longer_be_exported"),
+      "span_begin_line": Uint64(7),
+      "span_filename": String("src/lib.rs"),
+    },
```

Comment:
```suggestion
    },
    {
      "name": String("became_doc_hidden"),
      "span_begin_line": Uint64(16),
      "span_filename": String("src/lib.rs"),
    },
```

Reply:
Oops, I messed up the lint definition myself — I shouldn't code while jetlagged 😅 This suggestion is backwards, and this line should be reported in the lint about `#[doc(hidden)]`, not here.